main.rb

```diff
@@ -1,4 +1,5 @@
 require_relative 'app'
+require_relative 'manage_data'
 
```

Comment:
- [ ] While running `ruby main.rb` I am getting the error ``require_relative': cannot load such file --` what you can do is that please remove `require_relative 'manage_data'` form main.rb.